### DISCUSSION TITLE
form-group row: label and input are placed above each other when window at smallest

### DISCUSSION THREAD

**Discussion by HansKappert** [twbs/bootstrap#39535] [Q&A]
I created a minimal html using the bootstrap documentation to show that the label and input field are on the same line....until you make the browser window very small, like you would have it on a mobile phone. At that point the label and input field are placed on top of each other. 

Is that intended behaviour?

```html
<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"  integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <title>Title</title>
</head>

<body>
<form><div class="form-group row">
    <label for="inputEmail3" class="col-sm-2 col-form-label">Email</label>
    <div class="col-sm-6">
      <input type="email" class="form-control" id="inputEmail3" placeholder="Email">
    </div>
  </div>
</form>
</body>

</html>
```

**Comment by florianlacreuse** [comment]
Hi,

Are you familiar with the following concepts: [breakpoints](https://getbootstrap.com/docs/5.3/layout/breakpoints/) and [grid responsive columns](https://getbootstrap.com/docs/5.3/layout/grid/#responsive-classes)?

In your example there is a `.col-sm-6` class on a `<div>` tag. So under the `sm` breakpoint, this `<div>` tag will be like any other html block elements.

Note: You are using BS4, you should use BS5 if possible.
